### PR TITLE
Azure Event Grid: Creating baseline version for new API version 2017-09-15-preview

### PR DESCRIPTION
**Pull request by kalyanaj**
Azure Event Grid: Creating baseline version for new API version 2017-09-15-preview. 

1) With the first commit, the contents of this file are EXACTLY the same as the contents of 2017-06-15-preview with the exception of updating the API version from 2017-06-15-preview to 2017-09-15-preview. 
2) The follow-up commit contains the actual changes corresponding to the new API version.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sarangan12** [comment]
@kalyanaj
Can you fix this error:

{
  "type": "Error",
  "code": "XmsExamplesRequired",
  "message": "Please provide x-ms-examples describing minimum/maximum property set for response/request payloads for operations.",
  "id": "R2022",
  "validationCategory": "SDKViolation",
  "providerNamespace": null,
  "resourceType": null,
  "sources": [
    "file:///home/travis/build/Azure/azure-rest-api-specs/specification/eventgrid/resource-manager/Microsoft.EventGrid/2017-09-15-preview/EventGrid.json:18:4 ($.paths)"
  ]
}

**Comment by kalyanaj** [comment]
@sarangan12, Regarding the x-ms-examples, there's another active PR in review (#1648) to add x-ms-examples for Azure Event Grid (I had gotten an exception for this earlier). Once that PR gets merged, I will merge the x-ms-examples into the Swagger for the newer API version. The Swagger for the newer API version (2017-09-15-preview) is required to unblock the PRs for the .NET SDK / PowerShell etc., so it will be great if we can do them in parallel without blocking on it.

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**
